location-models/src/main/java/com/telecominfraproject/wlan/location/models/events/LocationChangedApImpactingEvent.java

```diff
@@ -0,0 +1,19 @@
+package com.telecominfraproject.wlan.location.models.events;
+
+import com.telecominfraproject.wlan.location.models.Location;
+
+public class LocationChangedApImpactingEvent extends LocationChangedEvent {
+	private static final long serialVersionUID = -5258621806247788808L;
+
+	public LocationChangedApImpactingEvent(Location location){
+        super(location);
+    }
+    
+    /**
+     * Constructor used by JSON
+     */
+    public LocationChangedApImpactingEvent() {
+        super(null);
```

Comment:
This is going to call the super constructor, below, and the dereferences on location will make an NPE. Please  fix.
    public LocationChangedEvent(Location location){
        **super(location.getCustomerId(), location.getLastModifiedTimestamp(), location);**
    }

Reply:
@mikehansen1970 , thank you for comments. I just made a new change, to make LocationChangedApImpactingEvent not an instance of LocationChangedEvent any more, so CMAP will get two events for LocationChangedEvent handling.
With the new change, the new constructor will be a standard one, like others, which only used for JSON.